tests/installation/partitioning_raid.pm

```diff
@@ -172,11 +172,17 @@ sub set_lvm {
     wait_still_screen;
 
     # create logical volume
-    send_key "alt-d";
-    send_key "down";
-    send_key "down";
-    send_key "ret";
-
+    if ($older_product) {
+        send_key "alt-d";
+        send_key "down";
+        send_key "down";
```

Comment:
Will it help to use `send_key_until` here?

Reply:
might be but who cares about older products? ;) At least I guess it should not be done in the same commit. Even we could merge this PR to provide the fix faster and then think about optimizing the robustness

Reply:
Yes, we have raised this point. We are maintaining 3 code flows now, which can become another mirror of main.pm issues =)

Reply:
Ah, I've missed that it's old code.